.github/workflows/scripts/build_pr.sh

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env bash
```

Comment:
This is a DRY version of the PR build script. It could theoretically be run manually but you would have to set up all the environment variables... fun!